utils/measure/.VERSION

```diff
@@ -1 +1 @@
-v0.13.2
+
```

Comment:
Is this correct? Version is empty?

Reply:
Good catch. This is not correct. I suspect it's an issue in the github action. Will have a look into that.